src/init.cpp

```diff
@@ -1256,29 +1241,26 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
 
     // ********************************************************* Step 5: Backup wallet and verify wallet database integrity
 #ifdef ENABLE_WALLET
-    if (!fDisableWallet) {
-        std::string strWarning;
-        std::string strError;
-
-        nWalletBackups = GetArg("-createwalletbackups", 10);
-        nWalletBackups = std::max(0, std::min(10, nWalletBackups));
+    std::string strWarning;
```

Comment:
We can't just throw away `!fDisableWallet` here - IMO this autobackup part (L1244-L1257) should be wrapped into a separate function in CWallet which should start with smth like `if (GetBoolArg("-disablewallet", DEFAULT_DISABLE_WALLET)) return true;` (similar to other functions modified by 06f41f35860ffb255c195b1d9a542b6f0a8392fa `init: Get rid of fDisableWallet`) to preserve existing logic and follow the idea of newly introduced changes at the same time.

Reply:
Agree and updated PR. I left `keePassInt.init()` untouched because it doesn't do any logic beside getting some value from command line.